tube/etl/outputs/es/writer.py

```diff
@@ -111,7 +139,7 @@ def create_guppy_array_config(self, etl_index_name, types):
         except Exception as e:
             print(e)
 
-    def write_df(self, df, index, doc_type, types):
+    def write_rdd(self, df, index, doc_type, types):
```

Comment:
the call to this function is still using the old name https://github.com/uc-cdis/tube/blob/fa251d3fd712b0b9b55cda21c3a4366b2c54ffcd/tube/etl/indexers/base/translator.py#L162

Reply:
we have different function called `write_df`